src/controller/java/JniReferences.cpp

```diff
@@ -68,6 +68,21 @@ CHIP_ERROR GetClassRef(JNIEnv * env, const char * clsType, jclass & outCls)
     return err;
 }
 
+CHIP_ERROR N2J_ByteArray(JNIEnv * env, const uint8_t * inArray, uint32_t inArrayLen, jbyteArray & outArray)
+{
+    CHIP_ERROR err = CHIP_NO_ERROR;
+
+    outArray = env->NewByteArray((int) inArrayLen);
+    VerifyOrExit(outArray != NULL, err = CHIP_ERROR_NO_MEMORY);
```

Comment:
Prefer to use return macros instead of `goto exit` macros, especially since exit here has no cleanup state.

Reply:
Done